skt/runner.py

```diff
@@ -528,15 +531,15 @@ def run(self, url, release, wait=False, host=None, uid="",
                         different host), False otherwise.
 
         Returns:
-            0 if everything passed
-            1 if testing failed
-            2 in case of infrastructure error (exceptions are logged)
+            SKT_SUCCESS if everything passed
+            SKT_FAIL if testing failed
+            SKT_ERROR in case of infrastructure error (exceptions are logged)
         """
-        ret = 0
+        ret = SKT_SUCCESS
         self.failures = {}
         self.recipes = set()
         self.watchlist = set()
-        self.aborted_count = 0
+        self.aborted_count = SKT_SUCCESS
```

Comment:
As the variable name suggests, the variable contains a count of aborted jobs, not a return code